providers/shared/outputhandlers/write_to_cmdb/outputhandler.ftl

```diff
@@ -12,29 +12,60 @@
 
 [#function shared_outputhandler_write_to_cmdb properties content ]
 
-    [#local fileProperties = getOutputFileProperties() ]
-    [#local fileFormat = (fileProperties["format"])!""]
-
-    [#-- An empty JSON object is considered empty content --]
-    [#-- Even if the content is empty we still write an empty file --]
-    [#if ! content?has_content ]
-        [#local content=""]
-        [#local fileFormat=""]
-    [/#if]
+    [#if properties["type"] == "file" ]
```

Comment:
Is this a transitional thing or for internal use only? We've discussed the engine owning the cmdb so if coming from an external trigger, then should be through a specific entrnace e.g. for updating a setting or build reference.

I guess the generation contract needs this option anyway ...

Reply:
This will always be used but more than likely moved into the cmdb provider as a handler that it owns 

With the logging support I think it highlights the need for non CMDB methods i.e. toFile cpFile etc.

Reply:
Agreed. I saw the new ticket for them.